src/utils/designConstants.js

```diff
@@ -0,0 +1,22 @@
+export const BREAKPOINTS = {
```

Comment:
This is an incremental step toward keeping our constants in one place. The long term plan is to have a `swarm-constants` module that provides vars/values in JS, Sass, and more